dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/PowerSystemStabilizers/Standard/PssKundur.mo

```diff
@@ -0,0 +1,69 @@
+within Dynawo.Electrical.Controls.Machines.PowerSystemStabilizers.Standard;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model PssKundur "Power system stabilizer based on Kundur's book"
+  import Modelica;
+  import Dynawo.Electrical.SystemBase;
+
+  parameter Types.PerUnit KStab "PSS gain";
+  parameter Types.Time t1 "Phase compensation time constant in s";
+  parameter Types.Time t2 "Phase compensation time constant in s";
+  parameter Types.Time tW "Washout time constant in s";
+  parameter Types.VoltageModulePu VsMinPu "Minimum output of PSS in pu (base UNom)";
+  parameter Types.VoltageModulePu VsMaxPu "Maximum output of PSS in pu (base UNom)";
+
+  //Input variables
+  Modelica.Blocks.Interfaces.RealInput omegaPu(start = SystemBase.omega0Pu) "Angular frequency in pu (base omegaNom)" annotation(
+    Placement(visible = true, transformation(origin = {-140, 0}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-46, -30}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput omegaRefPu(start = SystemBase.omega0Pu) "Reference angular frequency in pu (base omegaNom)" annotation(
```

Comment:
omegaRef0Pu, although it's the same value

Reply:
Is it actually a reference or the grid frequency ?

Reply:
Agreed and done for omegaRef0Pu.

I do not know, the answer may be in Kundur's book.

Reply:
The description is now
_"Grid angular frequency in pu (base omegaNom)"_